ui/components/SpinnerLoading/SpinnerLoading.styles.js

```diff
@@ -0,0 +1,237 @@
+import makeStyles from "@mui/styles/makeStyles";
+
+export const useStyles = makeStyles(() => ({
```

Comment:
Ideally we should be moving away from makeStyles, since we're using mui v5

Reply:
Agree 💯 👀 

But I guess we are using it in many places, so how about taking this as a new issue and updating it everywhere? It will be easier to track as well. Would love to know your opinion on this @Abhishek-kumar09 @warunicorn19 @sudo-NithishKarthik @Chadha93